test/Extensions/Tester.AzureUtils.Migration/OrleansTestSecrets.json

```diff
@@ -0,0 +1,5 @@
+{
+  "ClusteringConnectionString": "UseDevelopmentStorage=true",
```

Comment:
Should be removed, could potentially bit us in testing

Reply:
removed, thanks for spotting!